client/src/lib.rs

```diff
@@ -398,7 +397,7 @@ mod lib_tests {
 			band_url: "http://localhost:3000".to_string(),
 			domain: "0x0000000000000000000000000000000000000000".to_string(),
 			node_url: anvil.endpoint().to_string(),
-			verifier_address: format!("{:?}", verifier_address),
+			verifier_address: "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512".to_string(),
```

Comment:
We dont need verifier address anymore

Reply:
Got it, then I'll remove it from the config.